extends Node2D

var paused = false
var kiz_enter = false

var i = 0

var text1 = "Cevdet: Ben bir gÃ¶Ã§men kÄ±zÄ± gÃ¶rdÃ¼m Ihlara Vadisinde"
var text2 = "Fedora: Ne anam var ne babam. kalmÄ±ÅŸÄ±m Ã¶ksÃ¼z."
var text3 = "Cevdet: Sen bir Ã¶ksÃ¼z ben bir garip, alayÄ±m seni, alayÄ±m da gurbet elde sarayÄ±m seni."


func _ready() -> void:
	if Globals.otk:
		$Panel/Label/devam.text = "ð°“ð°€ð°‰ð°¢:ð°€ð±…ð°¢ð°€ð°š:ð°ƒð°²ð°¤:E:ð±ƒð°†ð±ð°£ð°€:ð°‰ð°€ð°½"
		text1 = "Cevdet: ð°‹ð°€ð°¤:ð°‹ð°ƒð°¼: ð°ð°‡ð°²ð°¢ð°€ð°¤: ð°¶ð°ƒð°”: ð°ð°‡ð°¼ð°“ð°‡ð°¢: ð°¶ð°žð°€ð°º: ð°‰ð°€ð°“ð°ƒð°¾ð°¦ð°€
"
		text2 = "Fedora: ð°¤ð°€: ð°£ð°¢: ð°‰ð°€ð°º: ð°¤ð°€: ð°‰ð°€ð°‰ð°¢: ð°´ð°€ð°žð°¢ð°ƒð±ð°¢: ð°œð°¾ð°‡ð°”
"
		text3 = "Cevdet: ð°¾ð°€ð°¤: ð°‹ð°ƒð°¼: ð°œð°¾ð°‡ð°”: ð°‹ð°€ð°¤: ð°‹ð°ƒð°¼: ð°ð°€ð°¼ð°ƒð°¯: ð°€ð°žð°–ð°ƒð°¢: ð°¾ð°€ð°¤ð°ƒ: ð°ð°†ð°ºð°‹ð°€ð±…: ð°€ð°¡: ð°½ð°€ð°ºð°–ð°ƒð°¢: ð°¾ð°€ð°¤ð°ƒ
"
		$son.text = "ð°½ð°†ð°£"
	Globals.set_music(load("res://kaynak/ses/Bolum3.mp3"))
	$son.hide()
func _process(delta: float) -> void:
	
	if not paused:
		if not kiz_enter:
			$CharacterBody2D.global_position.x += delta*40
		$Panel.hide()
	else:
		if Input.is_action_just_pressed("ui_attack"):
			print(i)
			if kiz_enter:
				if i == 2:
					$Panel/Label.text = text3
					$Panel.show()
					i+=1
					return
				$son.show()
			if i == 1:
				$gocmenKiz.scale.x *= -1
				
			paused = false
			$Panel/Label.text = text2
		

func _on_area_2d_body_entered(body: Node2D) -> void:
	
	
	
	if body.is_in_group("cicek"):
		$Panel.show()
		paused = true
		$cicek/cicek.hide()
		$Panel/Label.text = text1
		i+=1
	if body.is_in_group("kari"):
		$Panel.show()
		paused = true
		kiz_enter = true
		$Panel/Label.text = text2
		i+=1
	print(body)
